store/src/java/com/zimbra/cs/service/mail/SendMsg.java

```diff
@@ -285,6 +287,22 @@ public static ItemId doSendMessage(OperationContext oc, Mailbox mbox, MimeMessag
             ZimbraLog.soap.warn("Ignoring error while sending Calendar Invitation Forward Notification", e);
         }
 
+        //Log SENT event data
+        if(id != null) {
+            try {
+                Event sentEvent = new Event(mbox.getAccountId(), Event.EventType.SENT, System.currentTimeMillis());
+                sentEvent.setContextField(Event.EventContextField.SENDER, mm.getFrom()[0]);
+                sentEvent.setContextField(Event.EventContextField.MSG_ID, id);
+                for (Address address : mm.getAllRecipients()) {
+                    Event singleRecipientEvent = sentEvent.copy();
+                    singleRecipientEvent.setContextField(Event.EventContextField.RECEIVER, address.toString());
+                    EventLogger.getEventLogger().log(singleRecipientEvent);
+                }
+            } catch (MessagingException e) {
+                e.printStackTrace();
```

Comment:
We generally don't want to just print the stack trace of an error; in this case I would log a warning that the event could not be logged. Since the _event_ logger category will be used for successful events, this could go in the _soap_ or _misc_  categories.